src/main/java/seedu/student/model/student/VaccinationStatus.java

```diff
@@ -23,13 +23,12 @@
     public VaccinationStatus(String vaccinationStatus) {
         requireNonNull(vaccinationStatus);
         checkArgument(isValidStatus(vaccinationStatus), MESSAGE_CONSTRAINTS);
-        if (vaccinationStatus.equals("vaccinated")) {
+        if (vaccinationStatus.toLowerCase().equals("vaccinated")) {
             status = VaccinationStatusAbbreviation.VACCINATED;
-            textUI = vaccinationStatus;
         } else {
             status = VaccinationStatusAbbreviation.UNVACCINATED;
-            textUI = vaccinationStatus;
         }
+        textUI = status.toString().toLowerCase();
```

Comment:
is this field `textUI` used anywhere? it seems that no method uses it.

Reply:
It's being used in the UI side (`StudentCard.java`) in which the vaccination status label will be set to `textUI`.